site/src/components/Footer/footer.style.js

```diff
@@ -1,235 +1,328 @@
 import styled from "styled-components";
 
-export const Footerdiv = styled.footer`
-  display: flex;
-  flex-direction: column;
-  width: 100%;
-  height: auto;
-  background-color: black;
-  align-items: center;
-  padding-top: 3.5rem;
-  gap: 2.5rem;
-`;
+const FooterWrapper = styled.section`
+	padding: 2.5rem 3rem 1.5rem;
+	background: black;
```

Comment:
@Cvr421 I see you are using theme ref for colors on line 49. Good. Let's be consistent and use the theme here too.